src/widgets/vitals/vitals-biometrics-form/vitals-biometrics-form.utils.ts

```diff
@@ -0,0 +1,18 @@
+import { ConceptMetaData } from "./use-vitalsigns";
+
+export function calculateBMI(weight: number, height: number): number {
+  if (weight > 0 && height > 0) {
+    return Number((weight / (height / 100) ** 2).toFixed(1));
+  }
+}
+
+export const isInValidRange = (
+  conceptsMetaData: Array<ConceptMetaData>,
+  conceptUuid: string,
+  value: string | number
+) => {
+  if (value === undefined || value === "") return false;
+  const normalRange = conceptsMetaData.find(c => c.uuid === conceptUuid);
+  if (!normalRange?.lowNormal || !normalRange?.hiNormal) return false;
+  return !(value >= normalRange?.hiNormal && value <= normalRange?.lowNormal);
```

Comment:
Please write a test for this function. I do not think it works correctly. Might as well write one for `calculateBMI` too.

Reply:
Actually, instead of writing a test for this function, I suppose it would be better to write tests for VitalsAndBiometricForms which test the validation behavior. Per the advice "[avoid testing implementation details](https://kentcdodds.com/blog/testing-implementation-details)."